bridge/discord/discord.go

```diff
@@ -364,6 +398,21 @@ func (b *Bdiscord) replaceChannelMentions(text string) string {
 	return text
 }
 
+func (b *Bdiscord) replaceUserMentions(text string) string {
+	re := regexp.MustCompile("@[^ ]+")
```

Comment:
see somment about usernames or nicknames containing spaces, this is possible in discord iirc

Reply:
Alright, turns out that Discord usernames and nicks are limited to 32 characters. That means that the regex `(@[^@]{1,32})\b` would work to limit it to a max of 32 characters while automatically providing a word boundary to reduce the space a little further. From here, we can test each variation by popping off entire words at a time until we have a match or we've exhausted the testing space.

Reply:
That will give us a maximum of 16 tests in the worst case scenario.